site/zenodo_rdm/legacy/services.py

```diff
@@ -91,8 +92,44 @@ def create(self, identity, data, uow=None, expand=False):
             expand=True,
             uow=uow,
         )
+        communities = data.get("communities", [])
+        p = self.add_communities(communities, identity, uow, res._record)
+
+        # TODO how
+        # uow.register(IndexRefreshOp(indexer=current_record_communities_service.indexer))
+
         return res
 
+    def add_communities(self, communities, identity, uow, record):
+        """Creates community inclusion requests for a record's communities.
+
+        This bypasses the record communities service checks on community add.
+        """
+        if not communities:
+            return
+
+        processed = []
+        errored = []
+        for community in communities:
```

Comment:
This was taken directly from community's records service to bypass some validations.
It means that we can bypass this via config or similar.